create_aio_app/utils/config.py

```diff
@@ -11,7 +11,7 @@ def name_type(name: str) -> str:
 
     if not re.match(r"^[a-zA-Z0-9_]*$", name):
         raise argparse.ArgumentTypeError(
-            f"The format of the project name is incorrect."
```

Comment:
Why remove it?
Also, you could make the message useful by quoting the input and printing out expectations.

Reply:
I just removed f from beginning as it not need to use formatted string.

Reply:
It doesn't really add any value to the project, nor other benefits.

Reply:
What value should it add - it is just removed unnecessary character? i can return it back if you wish